### DISCUSSION TITLE
Save and preview

### DISCUSSION THREAD

**Discussion by sandyksit** [directus/directus#12675] [Archive]
Hi Team,

Is there any possibility unsaved data, can save first then preview the same data, 
I made button interface for 'preview' the collection.
But I want save unsaved data first then preview the same collection. that's mean when i clicked on 'preview' button it save the unsaved data first then preview the data. (I don't want click on 'save' /'save and stay' feature).

Thanks,
Sandeep

**Comment by rijkvanzanten** [comment]
If the item hasn't been saved yet, the locally made changes don't exist API side, so you won't be able to preview that information externally (as the data doesn't exist yet) 🤔 

The next best thing would be to save the record under a draft state, or with a published date in the future 👍🏻 (also see https://github.com/directus/directus/discussions/2975)
